zuchini-model/src/main/java/org/zuchini/parser/FeatureParser.java

```diff
@@ -69,13 +69,15 @@ static Feature parseWithWalker(ANTLRInputStream inputStream) {
     }
 
     static Feature parseWithVisitor(ANTLRInputStream inputStream) {
-        return newParser(inputStream).feature().accept(new Visitors.FeatureVisitor(inputStream.getSourceName()));
+        GherkinParser.FeatureContext feature = newParser(inputStream).feature();
+        return feature.accept(new Visitors.FeatureVisitor(inputStream.getSourceName()));
     }
 
     static GherkinParser newParser(ANTLRInputStream inputStream) {
         GherkinLexer lexer = new GherkinLexer(inputStream);
         GherkinParser parser = new GherkinParser(new BufferedTokenStream(lexer));
-        parser.setErrorHandler(new BailErrorStrategy());
+        parser.removeErrorListeners();
```

Comment:
Why remove the default (console) logger?

Reply:
The same message will be contained in the new exception, user should be responsible for logging or reacting on this.